tgui/packages/tgui/interfaces/AntagInfoMalf.tsx

```diff
@@ -85,9 +85,8 @@ const FlavorSection = (props, context) => {
           mt={-0.5}
           icon="hammer"
           tooltip={multiline`
-            This is a gameplay suggestion for bored ais.
-            You don't have to follow it, unless you want some
-            ideas for how to spend the round.`}
+            Please refer to the 'Antagonist Policy' section of the wiki
+            if you have any questions.`}
```

Comment:
GitHub won't let me do a code suggestion on this, but instead of having the original text deleted, comment the original tooltip out and make a new tooltip after with the Skyrat text we want. Additionally, put a comment in the code labeling it as a Skyrat edit. (Also if you do this you *probably* shouldn't need to have the Skyrat Edit comment at the top)